README.markdown

```diff
@@ -22,6 +24,9 @@ The projects follow the Liferay Workspace project style for gradle and maven.
 
 * `liferay-workspace` - in `gradle.properties`, Set `liferay.workspace.product`
   to the target product version.
+* `liferay-workspace-test-integration-sample` - You can find test project in
```

Comment:
@simonjhy , since this isn't merged yet, can we change the words just a little bit? 
`The test project is in modules/sample/sample-test folder...`

Reply:
yes, just named "integration-test-sample"?